.github/bin/init-cluster

```diff
@@ -31,7 +31,7 @@ options = {
   password: ENV.fetch("CB_PASSWORD", "password"),
   bucket: ENV.fetch("CB_BUCKET", "default"),
   sec_bucket: ENV.fetch("CB_SEC_BUCKET", ""),
-  server_quota: ENV.fetch("CB_SERVER_QUOTA", 2048).to_i,
```

Comment:
Why this change? We should probably also move to using dinocluster instead of this but that's a different topic...

Reply:
I need to hit the RAM quota limit to test the error that we get when trying to import a sample bucket with less that 200MB left. But I think I'm going to leave the quota and create some temp bucket to take up the space required.